### ISSUE TITLE
Decide how to handle schema

### ISSUE THREAD

**Issue by chrisdrackett** [yomajs/yoma#12]
Still getting my feet wet, but seems like we need to decide how to name and use schema. Now that I'm assuming we want this to come from nexus has does it integrate with Yoma?

**Comment by jasonkuhrt** [comment]
Nexus Framework used Nexus Schema as a library. Yoma can use it the same way. I think we should continue to do this. It worked well etc. before.

Any specific thoughts/questions?

**Comment by chrisdrackett** [comment]
nope! I just need to work through where the connections are. I'm also pretty far behind on nexus now as our app is still frozen at where things were before the framework went kaput, so I might have a bit of catch up to do :)

**Comment by jasonkuhrt** [comment]
@chrisdrackett I'm also in need of catch up :)
